Makefile

```diff
@@ -169,8 +170,15 @@ test-coverage: TESTARGS = -json -cover -covermode=atomic -coverprofile=coverage_
 test-coverage: test
 
 test-dependencies:
-	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
-	sudo mv /tmp/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH) /usr/local/kubebuilder
+	@if (ls $(KUBEBUILDER_DIR)/*); then \
+		echo "^^^ Files found in $(KUBEBUILDER_DIR). Skipping installation."; exit 1; \
+	else \
+		echo "^^^ Kubebuilder binaries not found. Installing Kubebuilder binaries."; \
+	fi
```

Comment:
Maybe move line 178-181 into `else` and remove `exit 1` looks more natural to me

Reply:
I thought about that, but with Make it would become one giant single line command and we'd lose the Makefile reporting on what line actually succeeded/failed.